explainer.md

```diff
@@ -186,225 +181,236 @@ https://user-images.githubusercontent.com/93594/141100217-ba1fa157-cd79-4a9d-b3b
+The pseudo-elements are constructed as documented in "how are transition elements represented".
 
-### CSS animation
+These pseudo-element selectors provide access to these pseudo-elements via `:root`:
 
-CSS can be used to build on default animations, or override the default.
+- `::page-transition(tag)` - Select the 'transition element' of a given `page-transition-tag`.
+- `::page-transition(tag)::image-wrapper` - Select the 'image wrapper'.
+- `::page-transition(tag)::image(outgoing | incoming)` - Select the incoming or outgoing image.
 
-```css
-::page-transition-container(header) {
-  /* … */
-}
-```
+`::page-transition` without a tag selects all transition elements.
 
-Element selectors:
+`::page-transition(tag nth odd)` can be used to select a subset of elements with the same tag, where 'odd' can be any valid [An+B](https://www.w3.org/TR/css-syntax-3/#anb-microsyntax).
 
-- `::page-transition-container(name)` - Select the transition containers of a given `page-transition-tag`.
-- `::page-transition-image-wrapper(page-transition-tag)` - Select the transition parts of a given `page-transition-tag`.
-- `::page-transition-outgoing-image(page-transition-tag)` - Select the outgoing image of a given `page-transition-tag`.
-- `::page-transition-incoming-image(page-transition-tag)` - Select the incoming image of a given `page-transition-tag`.
-- `::page-transition-root-outgoing` - Select the outgoing root image.
-- `::page-transition-root-incoming` - Select the incoming root image.
-- `::page-transition-root-container` - Useful for cases where something needs to be rendered underneath the root images.
+#### Default styles
 
-These will be selecting UA generated pseudo-elements.
+These styles will be in the UA stylesheet:
 
 ```css
-::page-transition-container(header) {
-  animation-delay: 300ms;
+::page-transition {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+::page-transition::image-wrapper {
+  position: absolute;
+  inset: 0;
+  isolation: isolate;
+}
+
+::page-transition::image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  /* This is a new proposed value. See 'cross-fading' */
+  mix-blend-mode: lighter;
 }
 ```
 
-CSS can be used to make changes to the default animation, or override `animation-name` to remove the default.
+#### Transition animations
 
-### JavaScript access to elements
+The default animations are as follows:
 
-So far the transition elements have been addressed by pseudo-element selectors, but JavaScript could be given access to the elements using the [CSSPseudoElement](https://drafts.csswg.org/css-pseudo-4/#CSSPseudoElement-interface) interface.
+- The `::image(incoming)` animates _from_ opacity 0 with a fill-mode of `both`.
+- The `::image(outgoing)` animates _to_ opacity 0 with a fill-mode of `both`.
+- If an element exists in both Page-A and Page-B, a transform animation takes its container from Page-A size/transform to Page-B size/transform.
```

Comment:
To clarify: if it only exists in one of the pages, it fades/appears in place?

But it can't happens unless at least one is offered up on both sides? :)

Reply:
> To clarify: if it only exists in one of the pages, it fades/appears in place?

Yeah, I'll clarify that

Reply:
This could be a UA detail but we'll need to do a width/height animation instead of changing size through a transform animation if the aspect ratio is different between the 2 images and we don't want the content to stretch as this ratio is changing.

Reply:
It definitely isn't a UA detail 😄 . Some thoughts on this https://github.com/WICG/shared-element-transitions/issues/108